src/run.ts

```diff
@@ -28,7 +28,8 @@ export async function run() {
       const resourceGroupName = core.getInput('resource-group', {
          required: true
       })
-      const clusterName = core.getInput('cluster-name', {required: true})
```

Comment:
recommend not removing `cluster-name` as that will impose burden on users when they upgrade the github action version.  If we really want to do this, we can do this in a separate PR.

Reply:
how about we keep `cluster-name` while also adding `resource-name`?  If `cluster-name` is supplied, we fall back to AKS behavior like before.  If it's not there, we use the new way with `resource-name` and `resource-type`.